src/common-principles.md

```diff
@@ -592,6 +593,20 @@ like in the example below.
 }
 ```
 
+The `ContextURI` can be used to provide a URL to a controlled vocabulary or ontology, or simply to provide a URL to a page that provides more context for the values in the column.
+It is to be used, so if combined with the value in that value (`<label>` or `<index>`), it would establish a unique identifier for the entity to be used across BIDS datasets.
+In the example below it would establish that `participant_id` is shared across datasets of the center, which would allow to differentiate center's subjects apart from subject ids used by other datasets used in the lab.
+
+```JSON
+{
+    "participant_id": {
+        "Description": "Unique identifier used for study subjects in the lab across studies",
+        "ContextURI": "https://thelab.example.com/term/subject/"
+    }
+}
+```
```

Comment:
I guess here would also be nice to give example with `bids:` URI to point to this particular dataset to say that IDs are unique to this dataset?